src/helpers/songs.ts

```diff
@@ -0,0 +1,86 @@
+import { getClient } from "../helpers";
+/*
+ **
+ **  Created By: niceEli & SPCFork
+ **  License: MIT
+ **
+ **  Used In KaboomTS Before NGJS
+ **
+ */
```

Comment:
Is the comment necessary? I can add you in the README if you want

Reply:
fixing rn

Reply:
fixed in 8e802578ff1e81b20a9dd528478a070298c4e490